user-manager-service/pom.xml

```diff
@@ -10,12 +10,13 @@
     </parent>
 
     <groupId>com.crm-food-establishment</groupId>
-    <artifactId>user-manager</artifactId>
```

Comment:
why did u make these changes to user-manager pom? =)

Reply:
done